### PR TITLE
Add placeholder property to mi-dropdown

### PR DESCRIPTION
**Pull request by matbmapspeople**
For PI improvement, I need a different text inside placeholder. To not break current interface, I made this implementation.

### CODE REVIEW

**Comment by matbmapspeople** [comment]
Do I need to change anything else to make it available to CMS components after releasing it to PROD? @ammapspeople

File: packages/components/src/components/dropdown/dropdown.tsx [hunk 1]
```diff
@@ -99,6 +99,14 @@ export class Dropdown {
      */
     @Prop() filterable = false;
 
+
+    /**
+     * If present, it dictates placeholder for an filterable input field in the dropdown. Defaults to 'Type to filter...'.
+     *
+     * @type {string}
+     */
+    @Prop() placeholder: string;
```
Comment (ammapspeople) [inline packages/components/src/components/dropdown/dropdown.tsx:108]: https://stenciljs.com/docs/properties#default-values

Looks like we can do 

```suggestion
    @Prop() placeholder: string = 'Type to filter...';
```

(not tested)

**Comment by ammapspeople** [comment]
> Do I need to change anything else to make it available to CMS components after releasing it to PROD? @ammapspeople

You'll need to update the components package in the CMS to the version in which this will be released.

File: packages/components/src/components/dropdown/dropdown.tsx (hunk 1 above)
Comment (matbmapspeople) [reply packages/components/src/components/dropdown/dropdown.tsx]: Tested, works. Suggestion applied.

**Review by ammapspeople — APPROVED** [review]
LGTM 😁

**Pull request merged**